data/presets/traffic_sign/hazard/horse_riders.json

```diff
@@ -0,0 +1,19 @@
+{
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "traffic_sign": "hazard",
+        "hazard": "horse_riders"
+    },
+    "reference": {
+        "key": "hazard",
+        "value": "horse_riders"
+    },
+    "aliases": [
+        "Equestrian Rider Sign",
+        "Horseback Rider Sign"
+    ],
+    "name": "Horse Rider Warning Sign"
```

Comment:
“Horse rider” sounds odd; “horseback rider” would be consistent with the various presets and fields that refer to “horseback riding”:

https://github.com/openstreetmap/id-tagging-schema/blob/0076d3bb97cc348a7f89d88ca6fa50b5aff76e8b/data/fields/horse_riding.json#L4